cmake/pi.cmake

```diff
@@ -26,14 +26,16 @@
 # https://github.com/Pro/raspi-toolchain/blob/master/Toolchain-rpi.cmake
 
 if("$ENV{RASPBERRY_VERSION}" STREQUAL "")
```

Comment:
Is this something set in bashrc? If so update documentation to include that. Otherwise I would recommend maybe having someone else test this to make sure it does not break things, but LGTM. Thanks for putting in the work on this.

Reply:
I was thinking the same thing. I tested it on our stuff, but not much in just kodlab world. I will do before merging. This is set in the BASHRC and it is defaulted to 1. I will add a change to the documentation.